protocol/src/main/resources/protocol.xml

```diff
@@ -9,10 +9,12 @@
       <type name="schemaId" primitiveType="uint16"/>
       <type name="version" primitiveType="uint16"/>
     </composite>
+
     <composite name="varDataEncoding">
-      <type name="length" primitiveType="uint16"/>
+      <type name="length" primitiveType="uint32" maxValue="2147483647"/>
```

Comment:
Why we have this thing so often :roll_eyes:

Reply:
Because we have different protocols. Any idea to avoid this?

Reply:
I think we could do this with xinclude

Reply:
Good idea. I opened a new issue: #3027

Reply:
Nice thanks